presto-hive/src/main/java/io/prestosql/plugin/hive/metastore/thrift/ThriftHiveMetastore.java

```diff
@@ -708,7 +734,7 @@ private void revokeRole(String role, String granteeName, PrincipalType granteeTy
                     .stopOn(UnknownDBException.class)
                     .stopOnIllegalExceptions()
                     .run("getAllViews", stats.getGetAllViews().wrap(
```

Comment:
Should this be `stats.getGetAllViews()` OR `stats. getGetTablesWithParameter` ?

Reply:
Good catch. Didn't really notice here. But after consideration I think keeping `stats.getGetAllViews()` should be better because it provides more detailed information for monitoring. @findepi Any comments?